Source/EPUBCore/FREpubParser.swift

```diff
@@ -21,15 +21,22 @@ class FREpubParser: NSObject, SSZipArchiveDelegate {
     Returns a FRBook.
     */
     func readEpub(epubPath withEpubPath: String) -> FRBook {
+        
         epubPathToRemove = withEpubPath
         
         // Unzip   
         let bookName = (withEpubPath as NSString).lastPathComponent
-        bookBasePath = (kApplicationDocumentsDirectory as NSString).stringByAppendingPathComponent(bookName)
-        SSZipArchive.unzipFileAtPath(withEpubPath, toDestination: bookBasePath, delegate: self)
+        bookBasePath = (kApplicationDocumentsDirectory as NSString).stringByAppendingPathComponent(bookName+".dir")
         
-        // Skip from backup this folder
-        addSkipBackupAttributeToItemAtURL(NSURL(fileURLWithPath: bookBasePath, isDirectory: true))
+        // unzip the epub if we have not already done so
+        var isDir : ObjCBool = true
+        if NSFileManager.defaultManager().fileExistsAtPath(bookBasePath, isDirectory: &isDir) == false {
```

Comment:
I don't think this is needed here, because I'm already checking it https://github.com/FolioReader/FolioReaderKit/blob/master/Source/FolioReaderContainer.swift#L111